client/signup/steps/user/style.scss

```diff
@@ -1,4 +1,5 @@
-body.is-section-signup.is-white-signup .signup.is-onboarding {
+body.is-section-signup.is-white-signup .signup.is-onboarding,
+body.is-section-signup.is-white-signup .signup.is-with-design-picker {
```

Comment:
Why do we need to apply these changes to the `user` step?

Reply:
Addressed in a call